cl_dll/StudioModelRenderer.cpp

```diff
@@ -349,13 +348,28 @@ void CStudioModelRenderer::StudioSlerpBones( vec4_t q1[], float pos1[][3], vec4_
 
 	s1 = 1.0f - s;
 
+	switch (m_pStudioHeader->numbones % 4)
```

Comment:
What if we have only more than 4 bones but not dividable by 4?

For example, 6? With this code, only first 4 bones will be interpolated.